packageurl_test.go

```diff
@@ -55,7 +55,7 @@ type OrderedMap struct {
 
 // qualifiersMapPattern is used to parse the TestFixture "qualifiers" field to
 // ensure that it's a json object.
-var qualifiersMapPattern = regexp.MustCompile(`^\{.*\}$`)
+var qualifiersMapPattern = regexp.MustCompile(`(?ms)^\{.*\}$`)
```

Comment:
`?s` will lead to matching new lines. It doesn't look like we did this before. Can you explain your intention behind this change? :)

Reply:
I‘ve described the intention n the PR description:

The only thing that does change to the test-suite is to use a multi-line matching for the JSON-data, so that not only `{"key": "value"}` qualifiers are valid, but also:
```json
{
  "key": "value"
}
```

(not that it's currently needed, but when I added tests my editor automatically made multi-line strings out of these, so I needed to fix it 🙂)